sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -1,3 +1,23 @@
+namespace Azure.Core
+{
+    public partial class JsonObjectSerializer : Azure.Core.ObjectSerializer
+    {
+        public JsonObjectSerializer() { }
+        public JsonObjectSerializer(System.Text.Json.JsonSerializerOptions options) { }
```

Comment:
I wonder if there's any reason we should consider abstracting away our own set of JsonSerializerOptions.   We can save that question until we figure out if/how we support other JSON serializers.

Reply:
There is a tentative plan to ship a Newtonsoft based implementation of this.